docs/debug.md

```diff
@@ -11,7 +11,7 @@ following command:
 ```
 $ sudo -s
 # ulimit -c unlimited
-# echo "/home/ubuntu/apps/hello-world.canonical/1.0.18/core.%e.%p.%h.%t" > /proc/sys/kernel/core_pattern
+# echo "/home/ubuntu/snap/hello-world.canonical/1.0.18/core.%e.%p.%h.%t" > /proc/sys/kernel/core_pattern
```

Comment:
I think the path is not 1.0.18, but a sequential identifier.
IMO, we are missing a snapcraft command that gives us the path to a snap data dir. Roight now, I'm not sure how to fix this particular case.